docs/dashboard/configuration/channels.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Channels
+---
+
+This page allows you to define and manage your sales channels for your store. You can create multiple channels with different currencies.
+
+![Channels](../screenshots/channels-main.jpg)
+
+## How to
+
+### How to create a channel
+
+1. The first step is to click the _CREATE CHANNEL_ button to create a channel in the _Channels_ section of dashboard configuration.
+
+2. The second step is to fill out the following information:
+   - Channel name
+   - Slug
+   - Channel currency
+
+   Once you have provided the above information, click the _SAVE_ button to save the channel.
+
+![New channel](../screenshots/channels-create-new.jpg)
+
+:::note Currency
+
+Once the channel has been created, you cannot change its currency.
+
+:::
+
+### How to activate a channel
+
+Newly created channels are inactive by default. Click the _ACTIVATE_ button to activate a channel. Likewise, to deactivate a channel, click the _DEACTIVATE_ button.
+
+![New channel](../screenshots/channels-status.jpg)
+
+### How to remove a channel
+
+You can remove a channel from the channel list view using the trash icon next to the channel's name.
+
+:::note Restrictions
+
+A channel can only be removed when it does not have any existing orders. If a channel has existing orders, you can only remove it once those orders have been migrated to another channel with the same currency.
```

Comment:
IMO description is confusing. If I were reading it the first time, I would start looking for Order Migration button.
Screenshot would be helpful too.
```suggestion
A channel can only be removed when it does not have any existing orders. If a channel has existing orders, you have to choose another channel with the same currency, where orders will be moved to.
```